roles/k3s_setup/tasks/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: Add IP address of all hosts to all hosts
+  lineinfile:
+    dest: /etc/hosts
+    regexp: '.*{{ item }}$'
+    line: "{{ hostvars[item].ansible_host }} {{item}}"
+    state: present
+  when: hostvars[item].ansible_host is defined
+  with_items: "{{ groups.all }}"
+
+- name: Copy k3s registries
+  ansible.builtin.template:
+    src: "registries.yaml.j2"
+    dest: "/etc/rancher/k3s/registries.yaml"
+    owner: root
+    group: root
+    mode: '0644'
+
+# TODO: Remove this once it exists in terraform
+- name: Add default route if in airgap setup
+  ansible.builtin.command: ip route add default via 192.168.201.1 dev eth0
```

Comment:
This metal lb IP range is hardcoded into too many places.

Please refactor `roles/metallb/defaults/main.yml` so that your plays and templates can leverage the [ipaddr](https://docs.ansible.com/ansible/latest/collections/ansible/utils/docsite/filters_ipaddr.html) filter. For example, this would be `metal_lb_ip_block | ansible.utils.ipaddr('first_usable')` if `metal_lb_ip_block` was `192.168.201.240/28`.